package.json

```diff
@@ -1,21 +1,24 @@
 {
-  "name": "websockets",
-  "version": "0.0.1",
-  "description": "client websocket for spiral framework",
+  "name": "@sf/websockets",
+  "version": "0.0.2",
+  "description": "Websocket client for spiral framework",
+  "main": "build/socket.js",
+  "types": "types/lib",
   "keywords": [
     "websocket",
     "client",
     "js"
   ],
   "scripts": {
+    "clean": "rimraf ./build && rimraf ./types",
+    "prebuild": "yarn clean",
     "type-check": "tsc --noEmit",
-    "type-check:watch": "npm run type-check -- --watch",
+    "type-check:watch": "yarn type-check -- --watch",
     "watch": "webpack --watch --colors",
     "build:js": "webpack",
-    "build:types": "tsc --emitDeclarationOnly",
-    "build": "npm run build:js && npm run build:types",
-    "start": "npm run watch && npm run type-check:watch",
-    "lint": "./node_modules/.bin/eslint ./**/*.ts --fix",
+    "build:types": "tsc --emitDeclarationOnly --outDir ./types",
+    "build": "yarn build:js && yarn build:types",
+    "lint": "eslint ./**/*.ts --fix",
```

Comment:
@Kreezag using 'node_modules' is bad practice as you are not guaranteed it exists